lib/chef_zero/server.rb

```diff
@@ -164,10 +165,11 @@ def port
     #
     def url
       sch = @options[:ssl] ? 'https' : 'http'
-      @url ||= if @options[:host].first.include?(':')
-                 URI("#{sch}://[#{@options[:host].first}]:#{port}").to_s
+      flattened_hosts = [@options[:host]].flatten
```

Comment:
Yeah, I can switch to that! I used the flatten method, because that was what was already in the code, but then I deleted it, so no reason to match it.